stl/inc/spanstream

```diff
@@ -226,7 +226,7 @@ public:
     // N4892 [ispanstream.assign], assignment and swap
     basic_ispanstream& operator=(const basic_ispanstream&) = delete;
 
-    basic_ispanstream& operator=(basic_ispanstream&& _Right) {
+    basic_ispanstream& operator=(basic_ispanstream&& _Right) noexcept {

```

Comment:
I think it's reasonable to strengthen exception specifications for move assignment operators.
However, you seemly forgot to do similar strengthening for `swap` functions (and other functions called by `swap`).